src/components/StoryView/StoryContainer.tsx

```diff
@@ -131,6 +133,22 @@ const StoryContainer = forwardRef<StoryRef, StoryContainerProps>(
                 }
                 onLongPress={onStoryPressHold}
                 onPressOut={onStoryPressRelease}>
+                {props.stories?.[progressIndex]?.showOverlay && (
+                  <View
+                    style={[
+                      styles.overlayViewStyle,
```

Comment:
@nilesh-simform Better to create a method that returns these styles and use them, even we can memorize this until we've a change in overlayViewPosition, also we can define enum for positions and use instead of using these directly